src/ngx_http_lua_shdict.c

```diff
@@ -654,7 +654,7 @@ ngx_http_lua_shared_dict_get(ngx_shm_zone_t *zone, u_char *key_data,
             return NGX_ERROR;
         }
 
-        ngx_memcpy(&value->value.b, data, len);
+        ngx_memcpy(&value->value.n, data, len);
```

Comment:
```suggestion
        value->value.n = *(double *) data;
```

Reply:
I believe it's better to use `ngx_memcpy(&value->value.n, data, len);`, since in pop helper and many other places, all SHDICT_TNUMBER type value are stored by ngx_memcpy. What do you think?

Reply:
This is not a big deal. This function is only used by the test cases.